Makefile

```diff
@@ -43,6 +43,30 @@ else ifeq ($(shell $(PKG_CONFIG) --exists libsystemd && echo "0" || echo "1"), 0
 	override CFLAGS += $(shell $(PKG_CONFIG) --cflags libsystemd) -D USE_JOURNALD=0
 endif
 
+define DOCKERFILE
+	FROM alpine:3.9
```

Comment:
nit of a nit and sorry that I didn't spot earlier.  Could we do `:latest` here for the tag?

Reply:
done